### PR TITLE
Add makeMention test

### PR DESCRIPTION
**Pull request by abtris**
- change structure
- add first 2 test
- add travici integration

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  var channel = slack.getChannelGroupOrDMByID(message.channel);
+  var user = slack.getUserByID(message.user);
+
+  if (message.type === 'message') {
+    var trimmedMessage = here.listen(message.text);
+    if (trimmedMessage !== null) {
+      var onlineUsers = getOnlineHumansForChannel(channel)
+          .filter(function(u) { return u.id != user.id; })
+          .map(function(u) { return here.makeMention(u.id); });
+      channel.send(onlineUsers.join(', ') + '\r\n' + user.real_name + ' said: ' + trimmedMessage);
```
Comment (pksunkara) [inline src/index.js]: Can we please change this to the following?

```js
channel.send(trimmedMessage + '\r\nSaid by @' + user.nick + ' /cc' + onlineUsers.join(', ')
```

**Comment by Almad** [comment]
@abtris given the size of the codebase and the usage, I think this is really a candidate to make it ES6 codebase...even without transpiling

plz

**Comment by abtris** [comment]
@almad ES6 isnt compatible with previous JS versions?

**Comment by abtris** [comment]
@pksunkara fixed

**Comment by pksunkara** [comment]
I can't merge. :disappointed: But :+1: to merge from me.

File: test/here_test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    assert.isNotNull(here.matchString('test string @here'));
+  });
+  it('should match @herebot message', function() {
+    assert.isNotNull(here.matchString('test string @herebot'));
+  });
+  it('should not match @here message', function() {
+    assert.isNull(here.matchString('`@here` test string'));
+  });
+  it('should not match @herebot message', function() {
+    assert.isNull(here.matchString('test string `@herebot`'));
```
Comment (kylef) [inline test/here_test.js]: Nice, I think we should also add a test for fenced code blocks too:

    ```
    @herebot
    ```

And perhaps block quotes?

```markdown
> @herebot test
```

**Pull request merged**